src/lib/types/experimental.ts

```diff
@@ -283,6 +284,10 @@ const flags: IFlags = {
         process.env.UNLEASH_EXPERIMENTAL_FLAG_OVERVIEW_REDESIGN,
         false,
     ),
+    showUserDeviceCount: parseEnvVarBoolean(
```

Comment:
seems to be inconsistent. We have user session count not device count in the code